### PR TITLE
GPII-2090: Perform a more fine grained selection of the files that need to be included into the installer

### PR DESCRIPTION
**Pull request by minogian**

### CODE REVIEW

**Comment by javihernandez** [comment]
ok to test

**Comment by minogian** [comment]
From https://ci.gpii.net/job/create-windows-vm/85/console:
> Build was aborted
> Aborted by anonymous
> Finished: ABORTED

Actually the build was almost over and without any problems up to that.
Can you try again?

**Comment by colinbdclark** [comment]
ok to test

**Comment by minogian** [comment]
Same error as before although earlier in the build this time (https://ci.gpii.net/job/create-windows-vm/86/console).
Perhaps there is a problem with the CI platform?

**Comment by colinbdclark** [comment]
@amatas, any thoughts on this?

**Comment by colinbdclark** [comment]
@minogian, do the tests pass on your local machine when you run them using the CI automation scripts?

**Comment by minogian** [comment]
@colinbdclark Yes they do.

Also, the same error appears while doing unrelated things (once while compiling the MSI file and once while cloning the installer repository). It seems to me its source is "external" to the provisioning scripts.

**Comment by amatas** [comment]
ok to test

**Comment by amatas** [comment]
ok to test

**Comment by amatas** [comment]
@colinbdclark @minogian it seems that the task in the CI took more time than expected while it updated some applications, and Jenkins stopped the build after 30 min. I've downloaded the latest Windows VM at the server that should save extra time because all the applications in the base image are installed and updated .

**Comment by minogian** [comment]
Rebased and resolved conflicts found after #93 was merged

File: provisioning/windows.rcj [hunk 1]
```diff
@@ -1,54 +1,41 @@
...
-:: Destination Directory :
-::
-    /DD:C:\installer\staging\windows\    :: Destination Directory.
-
 ::
 :: Include These Files :
 ::
-    /IF    :: Include Files matching these names
-::        *.*    :: Include all names (currently - Command Line may override)
+	/IF		:: Include Files matching these names
```
Comment (javihernandez) [inline provisioning/windows.rcj]: We tend to use tabs over spaces, can you please replace back all the tabs with spaces in this file?

**Comment by minogian** [comment]
@javihernandez OK. Pull request updated

**Comment by javihernandez** [comment]
Thanks @minogian!!

**Pull request merged**